diagnostic.md

```diff
@@ -94,4 +105,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+None of these will return a value. use the dot notation
+to determine hobby use 'teamMember.hobby'
+to determine name use 'teamMember.name'
```

Comment:
Understandably confusing, but the square brackets are not indicating an array in this case. Using square brackets is another way we can access the property value of an object. Dot notation is another "preferred" way. Also as a side note: to use dot notation to find the last value it would look like this: 
`teamMember.name.surname`
 The following are the answers.

```
 teamMember['hobby'] = 'exercise';
 
 teamMember['name'] = {
   given: 'Jason',
   surname: 'Weeks'
 }
 
 teamMember['name']['surname'] = 'Weeks';
 ```